apps/editors/helpers.py

```diff
@@ -964,3 +970,8 @@ def queue_tabnav_themes_interactive(context):
 @jinja2.contextfunction
 def is_expired_lock(context, lock):
     return lock.expiry < datetime.datetime.now()
+
+@register.function
```

Comment:
You don't need to add this. The registration decorator exposes the method to view templates which is not necessary here.